apps/admin_app/lib/admin_app_web/views/product_view.ex

```diff
@@ -95,7 +91,7 @@ defmodule AdminAppWeb.ProductView do
   end
 
   def get_product_display_image(product) do
-    product = Product.get_product_with_default_image(product)
+    # product = Product.get_product_with_default_image(product)
```

Comment:
Why is this commented? It preloads the image which is set as default in the product.